tests/integration/cattletest/core/test_authorization.py

```diff
@@ -1,6 +1,23 @@
 from common_fixtures import *  # NOQA
 
 
+def test_client_access(clients):
+    typesLen = {
+        'admin': 89,
+        'agent': 8,
+        'user': 68,
+        'agentRegister': 4,
+        'test': 140,
+        'readAdmin': 89,
+        'token': 2,
+        'superadmin': 140,
+        'service': 89,
+        'project': 68,
+    }
+    for tuple in clients.items():
+        assert typesLen[tuple[0]] == len(tuple[1].schema.types.items())
```

Comment:
Yeah.  I was just wondering whether we drone will spit out enough info to help with debugging.  Not a big deal since the dev can always just re-run the test locally and figure it out pretty quickly anyways.